front/app/containers/Admin/projects/project/phaseSetup/components/PhaseParticipationConfig/utils/participationMethodConfigs.ts

```diff
@@ -18,7 +18,7 @@ export const defaultParticipationConfig: IUpdatedPhaseProperties = {
   voting_term_singular_multiloc: null,
   voting_term_plural_multiloc: null,
   presentation_mode: 'card',
-  ideas_order: null,
+  ideas_order: undefined,
   input_term: 'idea',
```

Comment:
I think we might want to leave this as `null`, since the BE can be `nil` (`null`)

Reply:
See [answer](https://github.com/CitizenLabDotCo/citizenlab/pull/9352#discussion_r1841813775) to related question. :)